.ci/ci_cache_qemu.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+source "${cidir}/../lib/common.bash"
+source "${cidir}/lib.sh"
+
+WORKSPACE=${WORKSPACE:-$(pwd)}
+CURRENT_QEMU_VERSION=$(get_version "assets.hypervisor.qemu.version")
+QEMU_TAR="kata-qemu-static.tar.gz"
+
+cache_qemu_artifacts() {
+	local qemu_tar_location="$1"
+	[ -n "$qemu_tar_location" ] || die "couldn't retrieve Qemu location" 
```

Comment:
Nit: the correct name is "QEMU". See:

- https://github.com/kata-containers/tests/blob/master/cmd/check-spelling/data/acronyms.txt#L55
- https://en.wikipedia.org/wiki/QEMU

Reply:
Fixed.